runtime/tools/python/ttrt/__init__.py

```diff
@@ -5,19 +5,79 @@
 import ttrt.binary
 import os
 import json
-
+import importlib.machinery
+import sys
+import signal
+import os
+import io
+import subprocess
+import time
+import socket
+from loguru import logger
 from pkg_resources import get_distribution
-
-
-def system_desc_as_dict(desc):
-    return json.loads(desc.as_json())
-
+import sys
+import shutil
+
+import tracy
+import tracy_state
+from tt_metal.tools.profiler.process_ops_logs import process_ops
```

Comment:
I think these 3 imports will fail if you don't build with runtime enabled and then try to build/use ttrt.  Can you try building ttrt on macOS w/ this PR?

Reply:
I think one potential way to handle this, we can create our own perf wrapper module like `perf_trace.py` which includes these modules.  Then at this level we could do something like:
```
try:
    import perf_trace
except:
    perf_trace = None
```

Then down below in the `perf` function at the top:
```
if perf_trace is None:
    print("not supported blah blah blah")
    sys.exit(1)
```

Reply:
On this note, we should add ttrt build to CI, both for macOS and ubuntu